samples/bluetooth/alexa_gadget/README.rst

```diff
@@ -0,0 +1,166 @@
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set2_start
+   :end-before: set2_end
+
+The sample also requires:
+	
+* An Amazon Echo device
+* An Amazon developer account
+* A registered Gadget (see `Alexa Gadgets Registration`_)
+
+External dependencies
+=====================
+
+This sample has some specific external dependencies.
+
+Tools
+-----
+
+This sample depends on an external protobuf compiler tool - `Nanopb`_.
+
+This tool must be downloaded manually.
+
+Download Nanopb version **0.4.2** for your platform from the `Nanopb Downloads`_ site and extract the downloaded file to either this sample directory, or some other location where CMake can find it via a standard system PATH.
+
+Once the Nanopb tool has been downloaded, the build process is the same as for other samples.
+The tool is invoked automatically by CMake.
+
+Gadget registration
+-------------------
+
+When registering the Gadget, you will receive two identifier strings:
+
+1) Device type
+2) Device secret
+
+You must update the :option:`CONFIG_GADGETS_DEVICE_TYPE` and :option:`CONFIG_GADGETS_DEVICE_SECRET` configuration options in files :file:`prj.conf` and :file:`prj_nrf52dk_nrf52810.conf` with the values that you received during registration.
+
+User interface
+**************
+
+LED 1:
+   On when connected.
+
+LED 2:
+   On when handshake with peer is completed.
+
+LED 3:
+   On when wakeword directive (``Alexa.Gadget.StateListener``) is received.
+
+LED 4:
+   Toggles when Color Cycler skill is used.
+
+Button 4:
+   Erases bond information when button is held during power cycle or reset.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/alexa_gadget`
+
+.. include:: /includes/build_and_run.txt
+
+.. _peripheral_alexa_gadgets_testing:
+
+Testing
+=======
+
+After programming the sample to your board, test it by performing the following steps:
+
+1. Optionally, set up log monitoring:
+  a. Connect the board to the computer using a USB cable.
+     The board is assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.
+  b. |connect_terminal|
+2. Reset the board.
+3. Follow these instructions to pair your Echo device with the sample: `Alexa Gadgets Pairing`_.
+4. Observe that LED 1 turns on to indicate that a connection has been established.
+5. Observe that LED 2 turns on to indicate that the Alexa Gadgets handshake has completed.
+6. Speak the "Alexa" wakeword to your Echo device and observe that LED 3 turns on.
+7. If you are monitoring the log output from the COM port, observe log activity during Alexa queries.
+
+Dependencies
+************
+
+This sample uses the following |NCS| libraries:
+
+* :ref:`gadgets_service_readme`
+* :ref:`dk_buttons_and_leds_readme`
+* ``samples/bluetooth/alexa_gadget/src/gadgets/gadgets_profile.h``
```

Comment:
Not 100% sure we should be mentioning this, let me know what you think.

Reply:
As the profile is currently part of the sample, I'm fine either way here. If the profile was located elsewhere in the code base (like the gadget service currently is) I would have preferred to keep the reference

Reply:
After looking over it again, the wording should probably be changed if the gadgets_profile link should be kept.
Currently the documentation says "This sample uses the following |NCS| **libraries**:", which the profile currently is not.

Reply:
That's right, since this is strictly part of the sample and not a library, I think we can do without it here. Can you remove please?